# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/ScriptLoader.cpp
# Commit: 6125bb02952a
# Full Hash: 6125bb02952aff71c0d757e238a7752101b2d6c2
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-09-03 04:06:20
# Regressor Bug: 1556489
# File Overlap Count: 1
# Description:
#   Bug 1556489 - P23 - Only create new ClientSource objects in the content, but use the recreated info from redirects in the parent. r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D42260
# ==============================================================================

diff -r 06529d12b589 -r 6125bb02952a dom/workers/ScriptLoader.cpp
--- a/dom/workers/ScriptLoader.cpp	Mon Aug 26 12:18:57 2019 +1000
+++ b/dom/workers/ScriptLoader.cpp	Tue Aug 27 08:53:37 2019 +1000
@@ -1028,7 +1028,7 @@
       MOZ_DIAGNOSTIC_ASSERT(loadInfo.mReservedClientInfo.isSome());
       rv = AddClientChannelHelper(
           channel, std::move(loadInfo.mReservedClientInfo), Maybe<ClientInfo>(),
-          mWorkerPrivate->HybridEventTarget());
+          mWorkerPrivate->HybridEventTarget(), false);
       if (NS_WARN_IF(NS_FAILED(rv))) {
         return rv;
       }